ScnSoft-Education/LeetCodeTraining.Tests/TwoSumSolutionTest.cs

```diff
@@ -0,0 +1,27 @@
+using LeetCodeTraining.TwoSum1;
+
+namespace LeetCodeTraining.Tests
+{
+    public class TwoSumSolutionTest
+    {
+        [Fact]
+      
+            public void TwoSumArrayWith2Nums()
+            {
+
+                var nums = new int[] { 1, 2 };
+                int target = 3;
```

Comment:
Please use explicit variable naming everywhere or use var everywhere, not combining the variables declaration ways.